app/assets/javascripts/discourse/app/components/modal/create-invite.js

```diff
@@ -48,13 +48,15 @@ export default Component.extend(bufferedProperty("invite"), {
     this.set("topics", this.invite?.topics || this.model.topics || []);
 
     this.buffered.setProperties({
-      max_redemptions_allowed: 1,
-      expires_at: moment()
-        .add(this.siteSettings.invite_expiry_days, "days")
-        .format(FORMAT),
-      groupIds: this.model?.groupIds,
-      topicId: this.model.topicId,
-      topicTitle: this.model.topicTitle,
+      max_redemptions_allowed: this.model?.invite?.max_redemptions_allowed ?? 1,
+      expires_at:
+        this.model?.invite?.expires_at ??
+        moment()
+          .add(this.siteSettings.invite_expiry_days, "days")
+          .format(FORMAT),
+      groupIds: this.model?.invite?.groupIds,
+      topicId: this.model?.invite?.topicId,
+      topicTitle: this.model?.invite?.topicTitle,
```

Comment:
I see we access `this.model.invite` above so it's safe to assume `this.model` exists 😃

```suggestion
      max_redemptions_allowed: this.model.invite?.max_redemptions_allowed ?? 1,
      expires_at:
        this.model.invite?.expires_at ??
        moment()
          .add(this.siteSettings.invite_expiry_days, "days")
          .format(FORMAT),
      groupIds: this.model.invite?.groupIds,
      topicId: this.model.invite?.topicId,
      topicTitle: this.model.invite?.topicTitle,
```